setup.py

```diff
@@ -39,7 +39,7 @@
       scripts=glob('scripts/*py'),
       setup_requires=['numpy >= 1.9.2'],
       install_requires=[
-          'IPython < 4.0.0',
+          'IPython',
```

Comment:
Okay, I just tested it. And, you can start with an .md file, if thats what you prefer. I find the explicit conversion from ipynb to markdown easier... 

I think the main difference between 3.0 and 4.0 in getting the conversion working is which config file you modify. In 3.0, I think its and ipython config and in 4.0 its a jupyter config.